recipes/pngquant/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pngquant" %}
+{% set version = "2.13.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kornelski/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 2d462ca5739ac0afe5a6851914a48ff9d06d4fbd3c4883e8308de5dcc6d444bc
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/
```

```suggestion
  rpaths:
    - lib/
```

as above. If really needed, please add a comment why.